### PR TITLE
chore: Clean up tooling a bit

### PR DESCRIPTION
**Pull request by taion**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,36 +1,40 @@
...
-      python: "3.7"
+      python: "3.8"
       env: []
       before_install: skip
       install: skip
       script: skip
       deploy:
         provider: pypi
         user: itajaja
-        on:
```
Comment (itajaja) [inline .travis.yml]: double checking this is intentional

Comment (taion) [reply .travis.yml]: first commit was messed up; meant to add https://github.com/4Catalyzer/fourmat/pull/16/commits/f673492a794ad181dbf3866c91123713da615d21#diff-354f30a63fb0907d4ad57269548329e3R12

Comment (taion) [reply .travis.yml]: ah shoot this is necessary

**Comment by taion** [comment]
@itajaja btw you can use a token now if you want... probably better than putting your pypi PW in there

**Pull request merged**